src/v2/Apps/Gene/Components/GeneArtworkFilter.tsx

```diff
@@ -16,11 +36,44 @@ const GeneArtworkFilter: React.FC<GeneArtworkFilterProps> = ({
   relay,
 }) => {
   const { match } = useRouter()
+  const {
+    filtered_artworks: { counts },
+    sidebarAggregations,
+  } = gene
+
+  const { relayEnvironment, user } = useSystemContext()
+
+  const showNewFilters = getENV("ENABLE_NEW_ARTWORK_FILTERS")
```

Comment:
You can safely ignore this flag now. There's actually a ticket to get rid of it completely in the other components.

Reply:
👍 I removed the flag - now showing the new filter.